oem/azure/update-gallery.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script will update the Azure Gallery repo with the specified group and
+# version.
+
+set -e
+
+GALLERY_PATH="${1}/CoreOS"
+UGROUP="${2^}"
+LGROUP="${2,}"
+VERSION=$3
+
+MEDIA_PREFIX="2b171e93f07c4903bcad35bda10acf22"
+
+if [[ -z $GALLERY_PATH || -z $UGROUP || -z $VERSION ]]; then
+	echo "Usage: $0 <path to gallery repo> <group> <version>"
+	exit 2
+fi
+
+cd "${GALLERY_PATH}"
+git checkout -b "${LGROUP}-${VERSION}"
```

Comment:
maybe add `git fetch origin` and add `origin/master` to the end of that git checkout line. That way previous state won't matter much.